content/introduction/_index.md

```diff
@@ -3,16 +3,53 @@ title: "Introduction"
 weight: 1
 ---
 
-Welcome to Informo's specifications.
+### Welcome to Informo's specifications
 
 This documentation aims at giving the world a complete understanding of
 Informo's inner mechanisms, how it works and interacts with external elements in
-order to fulfil its mission. Informo needs to be as open and collaborative as
-possible for people to get involved, thus for it to work.
+order to fulfil its mission.
 
-For this reason, this documentation is also open and collaborative, which means
-that anyone can submit changes requests by following the [specifications change
-submission protocol](/introduction/scsp).
+Informo needs to be as open and collaborative as possible for people to get
+involved, thus for it to work. For this reason, this documentation is also open
+and collaborative, which means that anyone can submit changes requests by
+following the [specifications change submission protocol](/introduction/scsp).
+
+### What is Informo?
+
+Informo is a project aiming to fight online censorship of information by
+allowing information sources to share their news through a federation of servers
+(i.e. a group of servers that can talk to each other) using [the Matrix
+protocol](https://matrix.org). Matrix allows the information to be shared
+between a potentially large amount of servers, each storing their own copy and
+serving it to anyone requesting it.
+
+This significantly hampers censorship because:
+
+* every server knows about the information, the whole federation must be
+  blocked in order to censor it. The list of servers participating in a
+  federation can continuously grow and change, which can render such a blockade
+  practically very hard, if not impossible, to manage.
+
+* the Matrix protocol is built upon
+  [HTTP](https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol), which is
+  one of the most (if not the most) used protocols over the Internet, rendering
+  the identification of Matrix traffic (and therefore Informo traffic) hard (but
+  not impossible) to identify. On top of that, using security protocols such as
+  [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) would render
+  such identification even harder
+
+You can learn more about Informo's approach on [this dedicated
+page](/informo/what-is-informo/).
+
+The technical details for the distribution of information through the federation
+are described in [their own section](/information-distribution).
+
+On top of that, Informo also provides trust management mechanisms which are
+designed to let readers have a reliable idea of who an information source really
```

Comment:
I'd rather say `designed to let readers build a reliable source of information without having ...`

let me know what you think

Reply:
Err, users don't build a source of information, and I didn't really want to start mentioning trust networks from there. I think this fit the need of having a short description of trust management in one sentence without bringing up too many specific concepts.

Reply:
the part that bothers me is: "who an information source really is"

IMHO the identity of the source is not very relevant compared to the quality and reliability of its information.

Reply:
We don't provide a proof that "John Doe" is really named "John Doe" in real life. The trust network only helps certifying the trustworthiness of an information, not the source's owner identity.